scripts/create_users.go

```diff
@@ -0,0 +1,44 @@
+package scripts
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/Brightscout/mattermost-load-test-scripts/constants"
+	"github.com/Brightscout/mattermost-load-test-scripts/serializers"
+)
+
+func CreateUsers(config *serializers.Config) error {
+	client := model.NewAPIv4Client(config.ConnectionConfiguration.ServerURL)
+	var newUsers []*serializers.UserResponse
+	for _, user := range config.UsersConfiguration {
+		createdUser, cErr := client.CreateUser(&model.User{
+			Username: user.Username,
+			Email:    user.Email,
+			Password: user.Password,
+		})
+
+		if cErr.Error != nil {
+			return errors.New(cErr.Error.Message)
```

Comment:
can we use cErr.Error() here?

Reply:
no the type of `cErr.Error` is of the `*model.AppError` type